system/run-etcd.py

```diff
@@ -0,0 +1,175 @@
+#!/usr/bin/env python
```

Comment:
This script is almost identical to `system/run.py`:
```
/tmp/aptly etcd-dev ❯ diff system/run.py system/run-etcd.py
97a98,100
>                     t.configOverride = {
>                         "databaseEtcd": "127.0.0.1:2379"
>                     }
```
It would be much better to have a flag or an env variable to tell the test runner script to use etcd, rather than duplicate the whole script.

Moreover, I suspect that using `configOverride` may not work on all cases. Many unittests set their own overrides, so either they will be overwritten with the `databaseEtcd` key or viceversa.

Maybe `databaseEtcd` could come from the environment, e.g. `APTLY_ETCD_DATABASE` and set in `lib.py:BaseTest.config` dictionary?